zebra-rpc/src/methods.rs

```diff
@@ -109,9 +110,27 @@ pub trait Rpc {
     #[rpc(name = "getrawmempool")]
     fn get_raw_mempool(&self) -> BoxFuture<Result<Vec<String>>>;
 
+    /// Returns information about the given block's Sapling & Orchard tree state.
+    ///
+    /// zcashd reference: [`z_gettreestate`](https://zcash.github.io/rpc/z_gettreestate.html)
+    ///
+    /// # Parameters
+    ///
+    /// - `hash | height`: (string, required) The block hash or height. Height can
+    /// be negative where -1 is the last known valid block
+    ///
+    /// # Notes
+    ///
+    /// The negative height is not implemented yet.
+    // TODO: implement the negative heights and remove the note above.
```

Comment:
I think negative heights might be out of scope, because `lightwalletd` only uses positive heights above zero:
https://github.com/adityapk00/lightwalletd/blob/c1bab818a683e4de69cd952317000f9bb2932274/frontend/service.go#L340

I'll go update the ticket.

Reply:
I didn't know if we needed to support negative heights, but it seemed like a bit of extra work, so I thought it was worth checking the lightwalletd code.